uamqp/management_link.py

```diff
@@ -126,27 +128,39 @@ def _on_message_received(self, message):
                 to_remove_operation = operation
                 break
         if to_remove_operation:
-            mgmt_result = ManagementExecuteOperationResult.OK \
-                if 200 <= status_code <= 299 else ManagementExecuteOperationResult.FAILED_BAD_STATUS
             to_remove_operation.on_execute_operation_complete(
-                mgmt_result,
+                ManagementExecuteOperationResult.OK,
```

Comment:
So if the status code was an error code it would have been raised before this?

Reply:
nope, in uamqp we do `log.error()` if mgmt_result is not ok.
dealing with the status code is written at upper layer (e.g. service bus).

I will revert this.